production/helm/loki/templates/runtime-configmap.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "loki.fullname" . }}-runtime
```

Comment:
I think this should be the same as for the standard config:
https://github.com/grafana/loki/blob/c6f7a4122574752a95423075f9fdd5922db60ff9/production/helm/loki/templates/configmap.yaml#L5

Reply:
done :)